helm/docker-image/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG FROM_REPO=<myrepo>/temurin
+ARG FROM_TAG=jre-17
+
+FROM ${FROM_REPO}:${FROM_TAG} as builder
+
+ENV GATEWAY_VERSION 5
+ENV DOCKERIZE_VERSION v0.7.0
+
+USER root
+
+RUN set -eux && \
+    apt-get -qq update && apt-get --no-install-recommends --no-install-suggests -yqq install curl && \
+    curl --silent --location --retry 3 -o /tmp/gateway-ha.jar https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/$GATEWAY_VERSION/gateway-ha-$GATEWAY_VERSION-jar-with-dependencies.jar && \
+    curl --silent --location --retry 3 https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz | tar xz -C /usr/bin && \
+    curl --silent --location --retry 3 -o /tmp/gateway-ha-persistence-postgres.sql https://github.com/trinodb/trino-gateway/blob/main/gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql
+
+FROM ${FROM_REPO}:${FROM_TAG} as target
+
+ARG GATEWAY_UID=1001
+ARG GATEWAY_GID=1001
```

Comment:
```suggestion
ARG GATEWAY_GID=1000
```

Reply:
What is the reason for this change? Usually its considered more secure to run as UID 10001 or greater

Reply:
The file is already removed @willmostly